.gitignore

```diff
@@ -101,3 +101,4 @@ gmailLogin.properties
 webLogin.properties
 mailHandlerTestProps.properties
 config.yaml
+mturk.properties
```

Comment:
this one is just for the integration test, I dont want to use there the main config for.